vllm/attention/backends/flashinfer.py

```diff
@@ -141,23 +142,18 @@ def begin_forward(self):
             assert self.paged_kv_last_page_len is not None
             batch_size = self.query_start_loc.shape[0] - 1
             assert batch_size >= 0
-            # The profile run does not read kv cache.
-            # Both paged_kv_indices and paged_kv_last_page_len are empty.
-            # paged_kv_indptr is a zero tensor with size batch_size + 1.
-            if self.is_profile_run:
-                self.paged_kv_indptr = torch.zeros(batch_size + 1,
-                                                   device=self.device)
-            else:
+            # Only use flashinfer in the non-profile run
```

Comment:
This comment is a bit confusing as we are in the FlashInfer backend. Can we say more about why we skip the following logic in profile run and what's the outcome?